google-cloud-video-live_stream/samples/create_input.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def create_input project_id:, location:, input_id:
+  # [START livestream_create_input]
+  # project_id  = "YOUR-GOOGLE-CLOUD-PROJECT"  # (e.g. "my-project")
+  # location    = "YOUR-INPUT-LOCATION"  # (e.g. "us-central1")
+  # input_id    = "YOUR-INPUT"  # (e.g. "my-input")
+
+  # Require the Live Stream client library.
+  require "google/cloud/video/live_stream"
+
+  # Create a Live Stream client.
+  client = Google::Cloud::Video::LiveStream.livestream_service
+
+  # Build the resource name of the parent.
+  parent = client.location_path project: project_id, location: location
+
+  # Set the input fields.
+  new_input = {
+    type: Google::Cloud::Video::LiveStream::V1::Input::Type::RTMP_PUSH
+  }
+
+  operation = client.create_input parent: parent, input: new_input, input_id: input_id
+
+  # The returned object is of type Gapic::Operation. You can use this
+  # object to check the status of an operation, cancel it, or wait
+  # for results. Here is how to block until completion:
+  operation.wait_until_done!
+
+  # Print the input name.
+  puts "Input: #{operation.response.name}"
+  # [END livestream_create_input]
+
+  operation.response
+end
+
+
+if $PROGRAM_NAME == __FILE__
+  args    = ARGV.dup
+  command = args.shift
+
+  case command
+  when "create_input"
```

Comment:
we would not need the command to be separate as we have only one method each file.

Reply:
Please refer this PR https://github.com/googleapis/google-cloud-ruby/pull/19090/files#diff-c81b6c61172b0db53f3f62f36421248553ba8112b29e32b1f24813446a817312 we do not need this CLI

Reply:
Refactored